lib/chewy/query/criteria.rb

```diff
@@ -134,9 +134,9 @@ def storages
         STORAGES.map { |storage| send(storage) }
       end
 
-      def initialize_clone(other)
+      def initialize_clone(origin)
         STORAGES.each do |storage|
-          value = other.send(storage)
+          value = origin.send(storage)
```

Comment:
Yeah, it probably can, but I didn't configure rubocop to alert me yet :) Old habit, you know